lib/deployment.js

```diff
@@ -129,6 +127,13 @@ class Deployment extends EventEmitter {
     }, cb);
   }
 
+  // Produces an array that is the set difference between two arrays.
+  static _difference(arr1, arr2) {
+    const diff1 = arr1.filter(value => !arr2.includes(value));
+    const diff2 = arr2.filter(value => !arr1.includes(value));
```

Comment:
If we are trying to mimic the lodash's `_.difference` here, i don't think we need to find `diff2` and include it in the result. From what I understand, the result should just be the values from `arr1` that are not in `arr2`. Please correct me if i'm wrong.

Reply:
You're completely right. I missed the documentation and just assumed it was a set difference of the two arrays. So it could probably be a lot simpler. Thanks!

Reply:
FIxed :)